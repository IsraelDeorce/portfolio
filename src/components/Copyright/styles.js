import styled from 'styled-components'

export const Copyright = styled.div`
  display: flex;
  justify-content: center;
`
