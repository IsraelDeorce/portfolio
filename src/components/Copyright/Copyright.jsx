import { useTranslation } from 'react-i18next'
import { FaRegCopyright } from 'react-icons/fa'
import * as S from './styles'

function Copyright() {
  const { t } = useTranslation()
  return (
    <S.Copyright>
      Copyright&nbsp;
      <FaRegCopyright />
      &nbsp;{t('copyright')}
    </S.Copyright>
  )
}

export default Copyright
