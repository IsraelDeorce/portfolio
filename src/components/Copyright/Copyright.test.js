import Copyright from './Copyright'

describe('<Copyright/>', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Snapshot', () => {
    it('renders without crashing using shallow', () => {
      expect(shallow(<Copyright />)).toMatchSnapshot()
    })
  })
})
