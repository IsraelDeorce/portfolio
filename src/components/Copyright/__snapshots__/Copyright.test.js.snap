// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Copyright/> Snapshot renders without crashing using mount 1`] = `
<Copyright>
  <styled.div>
    <div
      className="sc-bdnxRM lilzio"
    >
      Copyright 
      <FaRegCopyright>
        <IconBase
          attr={
            Object {
              "viewBox": "0 0 512 512",
            }
          }
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm0 448c-110.532 0-200-89.451-200-200 0-110.531 89.451-200 200-200 110.532 0 200 89.451 200 200 0 110.532-89.451 200-200 200zm107.351-101.064c-9.614 9.712-45.53 41.396-104.065 41.396-82.43 0-140.484-61.425-140.484-141.567 0-79.152 60.275-139.401 139.762-139.401 55.531 0 88.738 26.62 97.593 34.779a11.965 11.965 0 0 1 1.936 15.322l-18.155 28.113c-3.841 5.95-11.966 7.282-17.499 2.921-8.595-6.776-31.814-22.538-61.708-22.538-48.303 0-77.916 35.33-77.916 80.082 0 41.589 26.888 83.692 78.277 83.692 32.657 0 56.843-19.039 65.726-27.225 5.27-4.857 13.596-4.039 17.82 1.738l19.865 27.17a11.947 11.947 0 0 1-1.152 15.518z"
              key="0"
            />
          </svg>
        </IconBase>
      </FaRegCopyright>
       
      copyright
    </div>
  </styled.div>
</Copyright>
`;

exports[`<Copyright/> Snapshot renders without crashing using render 1`] = `
<div
  class="sc-bdnxRM lilzio"
>
  Copyright 
  <svg
    fill="currentColor"
    height="1em"
    stroke="currentColor"
    stroke-width="0"
    viewBox="0 0 512 512"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm0 448c-110.532 0-200-89.451-200-200 0-110.531 89.451-200 200-200 110.532 0 200 89.451 200 200 0 110.532-89.451 200-200 200zm107.351-101.064c-9.614 9.712-45.53 41.396-104.065 41.396-82.43 0-140.484-61.425-140.484-141.567 0-79.152 60.275-139.401 139.762-139.401 55.531 0 88.738 26.62 97.593 34.779a11.965 11.965 0 0 1 1.936 15.322l-18.155 28.113c-3.841 5.95-11.966 7.282-17.499 2.921-8.595-6.776-31.814-22.538-61.708-22.538-48.303 0-77.916 35.33-77.916 80.082 0 41.589 26.888 83.692 78.277 83.692 32.657 0 56.843-19.039 65.726-27.225 5.27-4.857 13.596-4.039 17.82 1.738l19.865 27.17a11.947 11.947 0 0 1-1.152 15.518z"
    />
  </svg>
   copyright
</div>
`;

exports[`<Copyright/> Snapshot renders without crashing using shallow 1`] = `
<styled.div>
  Copyright 
  <FaRegCopyright />
   
  copyright
</styled.div>
`;
